src/SimpleThings/EntityAudit/Collection/AuditedCollection.php

```diff
@@ -514,8 +516,8 @@ protected function initialize()
 
             foreach ($rows as $row) {
                 $entity = array(
-                    'rev' => $row['rev'],
-                    'revtype' => $row['revtype']
+                    'rev'     => $row['rev'],
```

Comment:
please remove the unalignment from configuration

@see https://github.com/KnpLabs/KnpPaginatorBundle/pull/338#discussion-diff-42511167